docs/kafka/getting-started-kafka/README.adoc

```diff
@@ -127,24 +127,15 @@ ifndef::qs[]
 endif::[]
 
 .Procedure
-. In the {product-kafka} {service-url-kafka}[web console^], select *Kafka Instances* and click *Create Kafka instance*.
+. On the *Kafka Instances* page of the {product-kafka} web console, click *Create Kafka instance*.
+. Complete the *Create a Kafka instance* form to define the following instance details. Some values currently have only one option.
+* *Instance name*: Enter a unique name for the Kafka instance, such as `my-first-kafka-instance`.
+* *Cloud provider*: Accept the default value of *Amazon Web Services*. If you are creating a trial instance, you can select *Google Cloud Platform*.
```

Comment:
Suggest: "For a production Kafka instance, accept the default value of *Amazon Web Services*. If you're creating a trial instance, you also have the option to select *Google Cloud Platform*."